tests/bcl-test/BCLTests/common-SystemNetHttpTests.ignore

```diff
@@ -0,0 +1,32 @@
+# #1
+#  Expected: instance of <System.Net.Http.HttpRequestException>
+#  But was:  <System.Net.WebException>
+MonoTests.System.Net.Http.HttpClientTest.WildcardConnect
+MonoTests.System.Net.Http.HttpClientTest.Send_Transfer_Encoding_Custom
+
+#  Error Domain=NSURLErrorDomain Code=-1022 "The resource could not be loaded because the App Transport Security policy requires the use of a secure connection."
```

Comment:
You can fix this by adding this to the Info.plist for the app:

```xml
<key>NSAppTransportSecurity</key>
<dict>
    <key>NSAllowsArbitraryLoads</key>
     <true/>
</dict>
```

Reply:
Already dealt with :)